README.md

```diff
@@ -1,35 +1,47 @@
+The tool is accessed through the command `datawave`. There are 5 subcommands that provide access to different aspects of Datawave. For each of these, the `--help` option has been designed to be as self-explanatory as possible.
 
-The following is a list and quick description of each subcommand.
+The following is a list and brief description of each subcommand:
 * `datawave accumulo`: Refreshes or views the Accumulo cache.
 * `datawave authorization`: Returns the "whoami" information for the provided certificate.
-* `datawave dictionary`: Displays the columns for the datatypes in the database.
+* `datawave dictionary`: Displays the columns for the data types in the database.
 * `datawave ingest`: Interacts with the ingest portion of Datawave. It ingests a file if one is provided; otherwise, it displays current ingest jobs.
 * `datawave query`: Simplifies the query process with Datawave.
 
 ### Configuring Host and IP
-By default, all commands target the hostname ``. This can be overridden in two ways:
+By default, all commands will target an empty hostname and fail. This can be overridden in two ways:
 1) Use the `-u` or `--url` option to specify a different hostname for that specific run.
-1) Set the environment variable `DWV_URL` to always use a custom hostname. Unsetting this reverts to the default.
+2) Set the environment variable `DWV_URL` to always use a custom hostname. Unsetting this reverts to the default behavior.
 
-If you prefer to use an IP address (e.g., if DNS is unavailable), you can use the `-i` or `--ip` flag. This tells the tool to locate the IP based on Kubernetes pod information for the relevant service. In this case, you may also need to specify the namespace using the `-n` or `--namespace` option. If `-n` is omitted, the tool will attempt to retrieve the namespace from the environment variable `DWV_NAMESPACE`. If neither the option nor the environment variable is provided, the namespace defaults to `default`.
-
-**Note:** When using the `--ip` option, ensure that your Kubernetes configuration (`kubeconfig`) is properly set up and points to the correct cluster. Without a valid and properly configured `kubeconfig`, the tool will be unable to retrieve the necessary pod information for resolving the IP address.
+To use an IP address (e.g., if DNS is unavailable), the `-i` or `--ip` flag can be used. This instructs the tool to locate the IP based on Kubernetes pod information for the relevant service. In this case, it may also be necessary to specify the namespace using the `-n` or `--namespace` option. If `-n` is omitted, the tool will attempt to retrieve the namespace from the environment variable `DWV_NAMESPACE`. If neither the option nor the environment variable is provided, the namespace defaults to `default`.
```

Comment:
should also mention the `-l` or `--localhost` now. I should have updated the README for that... it enables running with localhost:<port>